.github/workflows/build-test.yml

```diff
@@ -187,7 +187,10 @@ jobs:
     name: .NET
     timeout-minutes: 20
     needs: smoke
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Windows build will fail at touch command, suggestion for `dotnet` workflow:
```
dotnet:
    name: .NET
    timeout-minutes: 20
    needs: smoke
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ windows-latest ]
    steps:
      - uses: actions/checkout@v3
      - name: Set up JDK11
        uses: actions/setup-java@v3
        with:
          java-version: '11'
          distribution: 'temurin'
      - name: Set up .NET 6.0.x
        uses: actions/setup-dotnet@v2
        with:
          dotnet-version: '6.0.x'
      - name: Create .glv files - Ubuntu
        if: contains(matrix.os, 'ubuntu')
        run: |
          touch gremlin-dotnet/src/.glv
          touch gremlin-dotnet/test/.glv
      - name: Create .glv files - Windows
        if: contains(matrix.os, 'windows')
        run: |
          echo $null >> gremlin-dotnet/src/.glv
          echo $null >> gremlin-dotnet/test/.glv
      - name: Build with Maven
        run: |
          mvn clean install -pl -:gremlin-javascript,-:gremlin-python,-:gremlint -q -DskipTests -Dci
          mvn verify -pl :gremlin-dotnet,:gremlin-dotnet-tests -P gremlin-dotnet -DincludeNeo4j
```

Reply:
It doesn't seem like it failed so it seems this might not be an issue after all. Hiding comment as it's resolved.